### DISCUSSION TITLE
请求增加剪切

### DISCUSSION THREAD

**Discussion by huangsha1** [shuax/MouseInc.Settings#423] [Ideas]
相比于复制，用的剪切更多一点

**Comment by shuax** [comment]
你自己设置随便改啊

**Reply by huangsha1** [reply]
咋改

**Reply by kevzcr** [reply]
> 咋改

![image](https://user-images.githubusercontent.com/88956040/191279541-83c16a9d-6d0c-48f7-be89-6ffcef5dc81b.png)

手势里改。

**Reply by huangsha1** [reply]
谢谢
